src/components/Card.js

```diff
@@ -0,0 +1,61 @@
+import * as React from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+
+import { PALETTE } from "../shared";
+
+import Text from "./Text";
+import SubsectionHeader from "./SubsectionHeader";
+
+const CardEl = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: 2rem;
+  border-radius: 4px;
+  border: solid 1px ${PALETTE.white80};
+  box-shadow: 0 8px 16px -8px rgba(19, 12, 51, 0.08);
+
+  ${SubsectionHeader} {
+    margin-bottom: 0.5rem;
+  }
+
+  img {
+    display: block;
+    overflow: hidden;
+    width: 3rem;
+    height: 3rem;
+    border-radius: 50%;
+    margin-bottom: 1.5rem;
+  }
+`;
+
+/**
+ * Note: This exports a React component instead of a styled-component.
+ * [Design Mockup](https://app.zeplin.io/project/5d5aecf2918cf74d46363015/screen/5d813f575d1398783f61df5b)
+ */
+
+const Card = ({ title, icon, children }) => (
+  <CardEl>
+    {icon && <img src={icon} alt={title} />}
+    <SubsectionHeader color={PALETTE.black60}>{title}</SubsectionHeader>
+    <Text>{children}</Text>
+  </CardEl>
+);
+
+Card.propTypes = {
+  /**
+   * The title of the card
+   */
+  title: PropTypes.string.isRequired,
```

Comment:
Nit: Since this is rendered as `{title}` under a styled component, better to type it as 
```suggestion
  title: PropTypes.node.isRequired,
```
for max flexibility.

Reply:
`<SubsectionHeader color={PALETTE.black60}>{title}</SubsectionHeader>` - even though SubsectionHeader is H3 tag?

Reply:
I agree with using PropTypes.node, the dev might want the flexibility to provide this as the contents:

```
<>
  <Icon /><StrongEl>Very</StrongEl> good title
</>
```

Usually, if something will be displayed in the DOM, you usually want to use PropTypes.node (unless, for example, it needs to be a string that's piped into an HTML attribute for accessibility reasons.)

Reply:
Q - Is the goal to allow flexibility to developers or follow the design system by a designer?

Reply:
Flexibility within constraints. My experience with design systems is that designs are rarely 100% within their constraints, and "that's not supported by the design system" isn't sufficient. I'm also pretty sure that `<Trans>some text</Trans>` comes through as a node, so I _think_ this'd warn about it.

Reply:
Good point